src/app/context/wallet/CosmosWalletProvider.tsx

```diff
@@ -94,7 +99,7 @@ export const CosmosWalletProvider = ({ children }: PropsWithChildren) => {
         });
       }
     },
-    [handleError, rpc],
+    [handleError],
```

Comment:
Don't we need to keep `rpc` in the dependency list given that it is still used on line 82?